### ISSUE TITLE
Branch not calculated for french iban

### ISSUE THREAD

**Issue by DaRochaRomain** [skwasjer/IbanNet#77]
Hello,

It seems that there is an issue parsing french iban. For exemple, if you try to parse this iban : 
```
var ibanParser = new IbanParser(IbanRegistry.Default);
var isValid = ibanParser.TryParse("FR7630001007941234567890185", out var iban);
var bank = iban.BankIdentifier;
var branch = iban.BranchIdentifier;
```
the BranchIdentifier is null instead of being 00794

Is there a way to fix that on my side?

**Comment by skwasjer** [comment]
It seems to not be part of the official SWIFT spec, but I don't know why. The registry is generated directly from that spec. Hence it is not really feasible to maintain a manual fix because it would be overwritten again the next time a new version of the spec is released.

You can however write a new registry provider (see `IIbanRegistryProvider`) which just declares the (adjusted) French registry definition and load it before loading the default `SwiftRegistryProvider` (https://github.com/skwasjer/IbanNet/wiki/Options#other-registry-providers) is loaded.

**Comment by skwasjer** [comment]
I will give it some more thought for a future version of IbanNet. A way to define 'unofficial fixes' could perhaps be an opt-in addition to the library.

**Comment by DaRochaRomain** [comment]
I don't realy get how it doesn't seems to be part of the swift spec, when your current pattern `"5!n5!n11!c2!n"` il already specifying 5 digits for the bank, 5 digits for the branch and 11 digits for the account number.

**Comment by skwasjer** [comment]
I don't know why I didn't think of it before, but it can be addressed at the code gen level, see PR #78. Let me know if this satisfies your need and I will merge and deploy the fix.
